templates/kubernetes/docs/install-offline.md

```diff
@@ -258,7 +258,7 @@ is covered in the [proxy documentation][].
 
 
 <!-- LINKS -->
-[LXD-image]: https://documentation.ubuntu.com/lxd/en/latest/image-handling
+[LXD-image]: https://linuxcontainers.org/lxd/docs/master/image-handling
```

Comment:
Looks like the docs have moved to documentation.ubuntu.com

Reply:
thanks for catching these